ci/test-coverage.sh

```diff
@@ -42,7 +42,9 @@ if [[ -z "$CODECOV_TOKEN" ]]; then
   echo CODECOV_TOKEN undefined, codecov.io upload skipped
 else
   # We normalize CI to `1`; but codecov expects it to be `true` to detect Buildkite...
-  CI=true bash <(curl -s https://codecov.io/bash) -X gcov -f target/cov/lcov.info
+  # Unfortunately, codecov.io fails sometimes:
+  #   curl: (7) Failed to connect to codecov.io port 443: Connection timed out
+  CI=true bash <(while ! curl -sS --retry 5 --retry-delay 2 --retry-connrefused https://codecov.io/bash; do sleep 10; done) -Z -X gcov -f target/cov/lcov.info
```

Comment:
[`-Z` is added...](https://github.com/codecov/codecov-bash/blob/585fef0b950c85f2b29de094aae58321b671a650/codecov#L143):

```
    -Z           Exit with 1 if not successful. Default will Exit with 0
```